templates/buildkite-elastic.yml

```diff
@@ -71,6 +72,11 @@ Parameters:
     Type: String
     Default: elastic
 
+  BuildkiteAgentsPerInstance:
+    Description: How Many Buildkite agents to run per instance/machine
```

Comment:
Perhaps we should put this param down with the autoscale rules?

* Number of agents per instances
* Min number of instances
* Max number of instances
* etc

Actually I can modify the param structuring once we merge it to master! Let's just get it working first :)